tests/config.spec.ts

```diff
@@ -0,0 +1,26 @@
+import * as m from 'mochainon';
+import * as path from 'path';
+
+import config from '../lib/config';
+
+// tslint:disable no-unused-expression
+
+describe('Config:', () => {
+	describe('.paths', () => {
+		describe('.user', () => {
+			it('should be an absolute path', () => {
+				const isAbsolute =
+					config.paths.user === path.resolve(config.paths.user);
+				m.chai.expect(isAbsolute).to.be.true;
```

Comment:
I didn't make any changes to the tests to ensure I keep my code compatible with the old tests